test/sql/updates/setup.continuous_aggs.v2.sql

```diff
@@ -132,15 +135,27 @@ SELECT generate_series('2018-11-01 00:00'::timestamp, '2018-12-15 00:00'::timest
 \if :has_refresh_mat_view
       GROUP BY bucket, location
       HAVING min(location) >= 'NYC' and avg(temperature) > 2;
+
     -- ALTER VIEW cannot rename columns before PG13, but ALTER TABLE
     -- works for views.
-    ALTER TABLE rename_cols RENAME COLUMN bucket to "time";
+    --
+    -- Renaming one column that is used in the group by and one that is
+    -- not in the group by. Both should work.
+    ALTER TABLE rename_cols RENAME COLUMN bucket TO "time";
+\if :has_cagg_rename_col_bug == false
+    ALTER TABLE rename_cols RENAME COLUMN humidity TO moisture;
+\endif
 \else
       GROUP BY bucket, location
       HAVING min(location) >= 'NYC' and avg(temperature) > 2 WITH NO DATA;
     SELECT add_continuous_aggregate_policy('mat_before', NULL, '-30 days'::interval, '336 h');
 
-    ALTER MATERIALIZED VIEW rename_cols RENAME COLUMN bucket to "time";
+    -- Renaming one column that is used in the group by and one that
+    -- is not in the group by. Both should work.
+    ALTER MATERIALIZED VIEW rename_cols RENAME COLUMN bucket TO "time";
+\if :has_cagg_rename_col_bug == false
+    ALTER MATERIALIZED VIEW rename_cols RENAME COLUMN humidity TO moisture;
+\endif
```

Comment:
Note: the 2.5.x series has a bug that prevents renaming column that are not in the `GROUP BY` clause, so we do not test to rename the columns for this version. It does not pose an upgrade problem since users cannot even rename the columns for this version.